tests/test_dateTime.py

```diff
@@ -395,6 +396,26 @@ def testPickle(self):
         nts = pickle.loads(pickle.dumps(ts))
         self.assertEqual(nts.nsecs(DateTime.UTC), int(1192755473000000000))
 
+    def testToAstropy(self):
+        """Test that DateTime converstion to astropy is exact to within machine
+        precision.
+        """
+        date = DateTime(0)
+        astropyTime = date.toAstropy()
+        self.assertFloatsAlmostEqual(date.get(), astropyTime.to_value(format="mjd"))
```

Comment:
Why is this needed instead of `self.assertAlmostEqual` ?  Aren't these all scalars?

Reply:
assertAlmostEqual uses a quite imprecise default and checks "decimal places" not "significant digits". Our `assertFloatsAlmostEqual` works for scalars as well, and defaults to machine precision.